README.md

```diff
@@ -17,9 +17,7 @@ is by using pip to pull it from PyPI:
 You may also use Git to clone the repository from
 GitHub and install it manually:
 
-    git clone https://github.com/tweepy/tweepy.git
-    cd tweepy
-    python setup.py install
+    pip install git+https://github.com/tweepy/tweepy.git
```

Comment:
That doesn't match the description anymore.
```suggestion
    git clone https://github.com/tweepy/tweepy.git
    cd tweepy
    pip install .
```

Reply:
This would literally negate the only change.

Reply:
The relevant part of the change is using pip to install the library rather than easy_install, which this still does.

Reply:
I thought that doing `pip install git+...` would use pip for it

Reply:
Yes, obviously, but again, that doesn't match the description for what this is supposed to do.

Reply:
I changed the content of the message.

Reply:
There's no reason to replace the instructions for cloning the repository.
You can simply add additional instructions for installing from GitHub without cloning the repository.